controllers/front/CartController.php

```diff
@@ -621,8 +657,12 @@ protected function areProductsAvailable()
 
             if ($currentProduct->hasAttributes() && $product['id_product_attribute'] === '0') {
```

Comment:
```suggestion
            if ($currentProduct->hasAttributes() && $product['id_product_attribute'] === 0) {
```
Fix for missing alert about simple product in the cart changed to product with attributes